kubernetes/structures_stateful_set.go

```diff
@@ -256,27 +259,6 @@ func patchStatefulSetSpec(d *schema.ResourceData) (PatchOperations, error) {
 	return ops, nil
 }
 
-func patchPodTemplateSpec(keyPrefix, pathPrefix string, d *schema.ResourceData) (PatchOperations, error) {
```

Comment:
Was this not behaving as expected?

Reply:
No, it used to selectively support updates to only a subset of pod template fields.

All fields in a pod template are updatable.

See also https://github.com/terraform-providers/terraform-provider-kubernetes/pull/389#issuecomment-484101413